### ISSUE TITLE
Some references prevent child class of SGTabbedPager to get collected by GC

### ISSUE THREAD

**Issue by OliverMDr** [Cheesebaron/SGTabbedPager#12]
The view that inherits from the SGTabbedPager class doesn't get disposed correctly when it disappears from the UI e.g. when the user navigates to another view. 

Apparently the reason are some references that prevent the GC to collect the view. With the following changes in SGTabbedPager.cs I could achieve that the view gets disposed correctly.

If desired I can create a pull request.

```
public override void ViewWillAppear(bool animated)
{
    base.ViewWillAppear(animated);

    this.TitleScrollView.Delegate = this;
    this.ContentScrollView.Delegate = this;

    ReloadData();          
}

public override void ViewWillDisappear(bool animated)
{
    base.ViewWillDisappear(animated);      

    foreach (var btn in this._tabButtons)
    {          
      btn.RemoveTarget(ReceivedButtonTab, UIControlEvent.TouchUpInside);   
    }

    this.TitleScrollView.Delegate = null;
    this.ContentScrollView.Delegate = null;
}
```

**Comment by Cheesebaron** [comment]
A PR would be very welcome 😃

**Comment by Cheesebaron** [comment]
This was fixed by #13
